routes/dms.js

```diff
@@ -195,8 +195,12 @@ module.exports = function () {
     // we can render as is in the template.
     datapackage.displayResources = await Promise.all(datapackage.displayResources.map(async item => {
       await Promise.all(item.resource.views.map(async (view, index) => {
-        if (view && view.specType === 'text' && item.resource.path) {
-          item.resource.views[index].content = await fetchTextContent(item.resource.path)
+        try {
+          if (view && view.specType === 'text' && item.resource.path) {
+            item.resource.views[index].content = await fetchTextContent(item.resource.path)
+          }
+        } catch(error) {
+          item.resource.views[index].content = 'There is no data for this view type'
```

Comment:
@gavram Maybe instead of `content`, sending it as `errorMessage` or `failReason` and then displaying it somehow different from a usual text view, so that it would be obvious to the user that this string is not the actual content, but an error warning?

Also, it's not certain here that there is no data, maybe there is data, but there was some other error that prevented from showing / fetching it. Maybe replacing it with something like 'Could not get data for this view'?

Reply:
@gavram I think saying "Could not get data for this view" is better as @EvgeniiaVak suggested. Could you change the message and we can get this PR merged?